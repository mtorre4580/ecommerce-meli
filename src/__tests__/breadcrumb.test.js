import React from 'react';
import Breadcrumb from '../components/commons/breadcrumb/breadcrumb';
import { mount } from 'enzyme';
import renderer from 'react-test-renderer';

describe('<Breadcrumb />', () => {

    const categories = [
        'Electrónica, Audio y Video',
        'Audio',
        'Reproductores Portátiles',
        'iPod',
        'iPod touch',
        '32 GB'
    ];

    it('should render snapshot', () => {
        const component = renderer.create(<Breadcrumb categories={categories} />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

    it('should be equal size categories props', () => {
        const wrapper = mount(<Breadcrumb categories={categories} />);
        expect(wrapper.props().categories).toEqual(categories);
    });
     
 });



