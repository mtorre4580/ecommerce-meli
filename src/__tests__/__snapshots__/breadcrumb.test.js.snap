// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Breadcrumb /> should render snapshot 1`] = `
<nav>
  <ol
    className="breadcrumb"
  >
    <li
      className="breadcrumb-item"
    >
      Electrónica, Audio y Video
    </li>
    <li
      className="breadcrumb-item"
    >
      Audio
    </li>
    <li
      className="breadcrumb-item"
    >
      Reproductores Portátiles
    </li>
    <li
      className="breadcrumb-item"
    >
      iPod
    </li>
    <li
      className="breadcrumb-item"
    >
      iPod touch
    </li>
    <li
      className="breadcrumb-item"
    >
      32 GB
    </li>
  </ol>
</nav>
`;
